frontend/src/components/Explanation.js

```diff
@@ -1 +1,20 @@
-// This is where the explanation component will be created
\ No newline at end of file
+// This is where the explanation component will be created
+
+
+import React from 'react';
+
+function ExplanationBox() {
+    return (
+        <div class="jumbotron jumbotron-fluid">
+        <div class="container">
+          <h1 class="display-4">What is this?</h1>
+          <p class="lead">This is a project powered by students that gives you the power to translate any news article to your preferred langauge :).</p>
+        </div>
+        <div style={{ display: "flex" }}>
```

Comment:
If you want some styling be sure to include a css file instead of inline styling. Its a little cleaner that way.